package.json

```diff
@@ -63,6 +63,7 @@
     "babel-loader": "8.0.5",
     "core-js": "2.6.9",
     "eslint": "5.16.0",
+    "eslint-plugin-jest": "^23.0.4",
```

Comment:
Please can you remove the `^`? It brings unexpected behaviour.

Reply:
Oh right! Good call.